CMakePresets.json

```diff
@@ -345,11 +391,21 @@
         "UBSAN_OPTIONS": "print_stacktrace=1:print_summary=1:halt_on_error=1:$penv{UBSAN_OPTIONS}"
       }
     },
+    {
+      "name": "Windows_Sanitizer_Experimental",
+      "inherits": "windows_base",
+      "configurePreset": "Windows_Sanitizer_Experimental"
```

Comment:
you're missing all the goodies that make ASAN and UBSAN useful in CI. Without halt_on_error=1, ubsan isn't deadly by default.

The env settings:
```json
      "environment": {
        "ASAN_OPTIONS": "strict_string_checks=1:check_initialization_order=1:strict_init_order=1:detect_stack_use_after_return=1:allocator_may_return_null=1:$penv{ASAN_OPTIONS}",
        "UBSAN_OPTIONS": "print_stacktrace=1:print_summary=1:halt_on_error=1:$penv{UBSAN_OPTIONS}"
      }
```

are not a unix-only thing.

Reply:
Note: You can even mark UBSAN deadly at compile time, significantly decreasing relocation amounts and binary size